moped-editor/src/views/projects/projectView/ProjectActivityLog.js

```diff
@@ -328,29 +340,20 @@ const ProjectActivityLog = () => {
                                                 changeItem.field,
                                                 String(changeItem.old)
                                               )
-                                            : lookupMap?.[change.record_type]?.[
-                                                changeItem.field
-                                              ]?.[changeItem.old] ||
-                                              String(changeItem.old)}
-                                          &quot;
-                                        </b>{" "}
-                                        to{" "}
+                                            : fieldFormat(changeItem.old)}
+                                        </b>
+                                        &nbsp;to&nbsp;
                                         <b>
-                                          &quot;
-                                          {isFieldMapped(
+                                        {isFieldMapped(
                                             change.record_type,
                                             changeItem.field
                                           )
                                             ? getMappedValue(
                                                 change.record_type,
                                                 changeItem.field,
-                                                String(changeItem.new)
+                                                String(changeItem.old)
```

Comment:
should this be `String(changeItem.new)` ?

Reply:
You were correct - I just tested this out on `test`.